erpnext/e_commerce/doctype/e_commerce_settings/e_commerce_settings.py

```diff
@@ -6,7 +6,7 @@
 from frappe.model.document import Document
 from frappe.utils import comma_and, flt, unique
 
-from erpnext.e_commerce.redisearch import (
+from erpnext.e_commerce.rsearch import (
```

Comment:
'redisearch_utils' or would be a better name for this IMO. 

It will make it clearer without opening the file what this file exactly contains.

Reply:
agree with @NagariaHussain it's also consistent with other util files 👍

Reply:
Agreed, have done that :smile: